sci-libs/libticalcs2/libticalcs2-1.1.9.ebuild

```diff
@@ -26,15 +27,19 @@ DEPEND="${RDEPEND}
 
 DOCS=( AUTHORS LOGO NEWS README ChangeLog docs/api.txt )
 
+src_prepare() {
+	default
+	eautoreconf
+}
+
 src_configure() {
 	econf \
 		--disable-rpath \
-		$(use_enable nls) \
-		$(use_enable static-libs static)
+		$(use_enable static-libs static) \
+		$(use_enable nls)
 }
 
 src_install() {
+	use doc && HTML_DOCS+=( docs/html/. )
 	default
-	use doc && dohtml docs/html/*
-	use static-libs || rm "${D}"/usr/$(get_libdir)/${PN}.la
```

Comment:
have you checked that `.la` files are removed/even necessary? https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Handling_Libtool_Archives